brambling/payment/tests/test_dwolla.py

```diff
@@ -51,7 +52,7 @@
 
 
 class DwollaChargeTestCase(TestCase):
-
+    @skipUnless(os.environ.get('DWOLLA_TEST_APPLICATION_KEY'), 'dwolla test settings required')
```

Comment:
Clarification: Is there something about the new setup that required you to add these skip decorators, or was this just to aid in local dev?

Reply:
This was added because these tests require API keys, which we don't have a secure way to share with folks now that we're going 12-factor & 100% open-source. We can still keep the test API keys locally for our own use and on travis for CI, but we need to skip them if they aren't present.

Reply:
As long as they're still running in CI, I'm happy.